### ISSUE TITLE
Payment - An incorrect bank error message displayed

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#12138]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR** https://github.com/Expensify/App/pull/11839

## Action Performed:
1. Go to URL https://staging.new.expensify.com/and login
2. Go to settings > Payments > Add payment method
3. Bank account > Plaid flow
4. While in Plaid Sandbox mode, enter any data
5. You'll be brought back to Expensify
6. Select a bank account in the dropdown
7. Use a wrong password > Click Save & continue
8. Verify the error says "An unexpected error occurred while trying to add your bank account. Please try again."

## Expected Result:
We should see "An unexpected error occurred while trying to add your bank account. Please try again" error message

## Actual Result:
An incorrect bank error message displayed

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.19.1

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/197916413-6c1061ca-b2ec-4c92-9598-c5358a6c7c60.mp4

![Screen Shot 2022-10-25 at 3 30 18 PM](https://user-images.githubusercontent.com/93399543/197916460-18c796cd-c692-4590-8f2f-fa1cf82f25b0.png)

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Puneet-here** [comment]
Hi, it's not a bug. Because of adding new validation you are seeing this error message which is expected. Now you won't see the error message mentioned at https://github.com/Expensify/App/pull/11839 unless the api fails.

**Comment by yuwenmemon** [comment]
Yes we just need to update the TestRail steps cc @Expensify/applauseleads 

`Incorrect Expensify password entered` is the expected error message now for those test steps.

Closing this out! Let me know if you need me to create a separate issue for updating the test steps.
